test/hybig/HyBIG_Regression.ipynb

```diff
@@ -388,13 +397,294 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05be0959",
+   "metadata": {},
+   "source": [
+    "### Test that specifies dimensions overrides GIBS-compatible defaults:\n",
+    "\n",
+    "This test will specify the size of the output browse image dimensions in the request. In this test, the output will be asked to be square. Because the input has twice as many pixels in the x-direction, the output browse image will look squashed in the horizontal direction."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ada810a9",
+   "id": "a3eae07c",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "if environment_information is not None:\n",
+    "    dimensions_reference_dir = reference_dir / 'dimensions'\n",
+    "\n",
+    "    dimensions_request = Request(collection=environment_information['measures_collection'],\n",
+    "                                 granule_id=environment_information['measures_granules'][0],\n",
+    "                                 height=180, width=180, format='image/png')\n",
+    "\n",
+    "    dimensions_job_id = harmony_client.submit(dimensions_request)\n",
+    "    harmony_client.wait_for_processing(dimensions_job_id, show_progress=True)\n",
+    "\n",
+    "    reference_files = build_file_list(measures_basename, dimensions_reference_dir, 'PNG')\n",
+    "\n",
+    "    with TemporaryDirectory() as temp_dir:\n",
+    "        downloaded_dimensions_outputs = [\n",
+    "            file_future.result()\n",
+    "            for file_future\n",
+    "            in harmony_client.download_all(dimensions_job_id, overwrite=True, directory=temp_dir)\n",
+    "        ]\n",
+    "\n",
+    "        test_files = build_file_list(measures_basename, Path(temp_dir), 'PNG')\n",
+    "\n",
+    "        for file_name in test_files:\n",
+    "            assert file_name.exists() == True, f'File does not exist {file_name.resolve()}'\n",
+    "\n",
+    "        print_success('all test files generated')\n",
+    "\n",
+    "        expected_output_metadata = {\n",
+    "            **common_measures_metadata,\n",
+    "            'driver': 'PNG',\n",
+    "            'count': 1,\n",
+    "            'transform': Affine(2.0, 0.0, -180.0, 0.0, -1.0, 90.0),\n",
+    "            'height': 180,\n",
+    "            'width': 180\n",
+    "        }\n",
+    "\n",
+    "        assert_dataset_produced_correct_results(\n",
+    "            test_files[0], expected_output_metadata, reference_files[0], 'PNG'\n",
+    "        )\n",
+    "\n",
+    "    print_success('Conversion of MEaSUREs GeoTIFF to PNG specifying dimensions. Success')\n",
+    "else:\n",
+    "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11639b67",
+   "metadata": {},
+   "source": [
+    "### Test of tiled outputs:\n",
```

Comment:
I should have thought of this for testing DAS-1833 - by forcing the resolution to be really fine with the Harmony message, this test gets us a nice couple of tiles. (And it looks like it's behaving, so that's a relief)